core/src/main/java/com/linecorp/armeria/server/Service.java

```diff
@@ -140,4 +140,19 @@ R decorate(Function<T, R> decorator) {
     default boolean shouldCachePath(String path, @Nullable String query, PathMapping pathMapping) {
         return pathMapping.exactPath().isPresent() && query == null;
     }
+
+    /**
+     * Invoked when this {@link Service} starts to process a {@link Request}.
+     */
+    default void startToProcessRequest(HttpRequest req,
```

Comment:
I feel, these method names and javadocs are not saying correct behavior but I could not think of better names. Could you give me suggestions?

Reply:
Agree that providing these as lifecycle methods seems unnecessary, we can have a method to provide a characteristic of the service

Do you think these requests should also be left out of e.g., server "total active requests"? If so then something like `boolean transientRequests()` maybe? The requests are there but transient and not included as server requests, similar to how transient fields are not included in serialization. Otherwise, more explicit would be `requestsNotActive()` or similar.

Reply:
Thanks for suggestions. Let me use a word, `transient` with `TransientService` idea from trustin.